src/pynwb/image.py

```diff
@@ -80,6 +81,10 @@ def __init__(self, **kwargs):
         for key, val in args_to_set.items():
             setattr(self, key, val)
 
+    @staticmethod
+    def _check_data_timestamps_mismatch(data, timestamps):
+        return not np.array_equal(data, ImageSeries.DEFAULT_DATA) and timestamps is not None
+
```

Comment:
@rly I have an issue with this, since this check is in the TimeSeries init, I'm not sure how 
to pass the external_file or format as kwargs. I noticed this TODO,
https://github.com/NeurodataWithoutBorders/pynwb/blob/a62c4e951df08e7fff0095789f244513b7d57a7c/src/pynwb/image.py#L76
which is connected to this warning I believe. So instead of overwriting that method with an additional check, I should ignore the warning when external file is provided within the ImageSeries init?

Reply:
does this method need to be static?